synfig-studio/src/gui/cellrenderer/cellrenderer_timetrack.cpp

```diff
@@ -529,9 +529,9 @@ CellRenderer_TimeTrack::activate_vfunc(
 		//Deal with time point selection, but only if they aren't involved in the insanity...
 		Time stime;
 		const Node::time_set *tset = get_times_from_vdesc(value_desc);
-		bool clickfound = tset && get_closest_time(*tset, actual_time*time_dilation + time_offset, waypoint_width_time, stime);
```

Comment:
I suggest moving the variable declaration here and do division in-place: 

```
	const int waypoint_width = cell_area.get_height();
	const Time waypoint_width_time = time_plot_data.get_delta_t_from_delta_pixel_coord(waypoint_width) / 2;
```

Reply:
Done :)